### PR TITLE
Update reamdme

### PR DESCRIPTION
**Pull request by pacmessica**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,35 +27,118 @@ To compile sources you need to have go 1.14.1+ installed.
...
+{
+    "address": "0.0.0.0",
+    "port": "3000",
+    "http_port":"8087",
+    "polkadot_proxy_addr":  "localhost:50051",
+    "network": "polkadot",
+    "chain_id": "mainnet",
+    "managers":"0.0.0.0:8085",
+    "currency": "DOT",
+    "exp": "10",
```
Comment (patiee) [inline README.md:40]: default is 12

Comment (pacmessica) [reply README.md:40]: it shows DOT as 10 here: https://wiki.polkadot.network/docs/en/learn-DOT

maybe we should change the default to 10 too?

Comment (pacmessica) [reply README.md:40]: @patiee wondering where you got default 12 from? Heard we restarted worker with 12 today, and amounts created from this were off when compared to subscan. When exp is set to 10, the amount we calculate is consistent with amounts we see on other polkadot explorers. So we restarted worker again with 10 😬 

compare these:
https://polkadot.subscan.io/extrinsic/0xf41a8e40d200f7b79e34239530aaad2d88156ce688c25f4226ff8bab5e1828bb

```
   {
        "id": "7d83ef9f-5298-474a-84bd-45f25b11acaa",
        "hash": "0xf41a8e40d200f7b79e34239530aaad2d88156ce688c25f4226ff8bab5e1828bb",
        "block_hash": "0xbe233da149a5b55dac8d9ec0dc52a9c0f94c021df401c490ff19e7daa65aa96e",
        "height": 4530163,
        "epoch": "310",
        "chain_id": "mainnet",
        "time": "2021-04-07T14:53:36Z",
        "transaction_fee": [
            {
                "text": "0.0156000016DOT",
                "currency": "DOT",
                "numeric": 156000016,
                "exp": 10
            }
        ],
        "version": "0.0.1",
        "events": [
            {
                "kind": "Extrinsic",
                "type": [
                    "transfer"
                ],
                "module": "balances",
                "sub": [
                    {
                        "id": "4530163-1",
                        "type": [
                            "killedaccount"
                        ],
                        "module": "system",
                        "node": {
                            "versions": [
                                {
                                    "id": "13n7RQQfwJ5iV34BjoFAi78Qt1RvN9pDuJBYDM1Fw5LihCDA"
                                }
                            ]
                        },
                        "nonce": "1",
                        "completion": "2021-04-07T14:53:36Z",
                        "additional": {
                            "attributes": [
                                "{\"name\": \"AccountId\", \"value\": \"13n7RQQfwJ5iV34BjoFAi78Qt1RvN9pDuJBYDM1Fw5LihCDA\"}"
                            ]
                        }
                    },
                    {
                        "id": "4530163-2",
                        "type": [
                            "dustlost"
                        ],
                        "module": "balances",
                        "node": {
                            "versions": [
                                {
                                    "id": "13n7RQQfwJ5iV34BjoFAi78Qt1RvN9pDuJBYDM1Fw5LihCDA"
                                }
                            ]
                        },
                        "nonce": "1",
                        "completion": "2021-04-07T14:53:36Z",
                        "amount": {
                            "0": {
                                "text": "0.0008999984DOT",
                                "currency": "DOT",
                                "numeric": 8999984,
                                "exp": 10
                            }
                        },
                        "additional": {
                            "attributes": [
                                "{\"name\": \"AccountId\", \"value\": \"13n7RQQfwJ5iV34BjoFAi78Qt1RvN9pDuJBYDM1Fw5LihCDA\"}",
                                "{\"name\": \"Balance\", \"value\": 8999984}"
                            ]
                        }
                    },
                    {
                        "id": "4530163-3",
                        "type": [
                            "transfer"
                        ],
                        "module": "balances",
                        "sender": [
                            {
                                "account": {
                                    "id": "13n7RQQfwJ5iV34BjoFAi78Qt1RvN9pDuJBYDM1Fw5LihCDA"
                                },
                                "amounts": [
                                    {
                                        "text": "28.2325781984DOT",
                                        "currency": "DOT",
                                        "numeric": 282325781984,
                                        "exp": 10
                                    }
                                ]
                            }
                        ],
                        "recipient": [
                            {
                                "account": {
                                    "id": "1PLqR46mAv2uRPbdNtXqmQ869TfwS1H7m9Ku6yMeV5isnCd"
                                },
                                "amounts": [
                                    {
                                        "text": "28.2325781984DOT",
                                        "currency": "DOT",
                                        "numeric": 282325781984,
                                        "exp": 10
                                    }
                                ]
                            }
                        ],
                        "node": {
                            "recipient": [
                                {
                                    "id": "1PLqR46mAv2uRPbdNtXqmQ869TfwS1H7m9Ku6yMeV5isnCd"
                                }
                            ],
                            "sender": [
                                {
                                    "id": "13n7RQQfwJ5iV34BjoFAi78Qt1RvN9pDuJBYDM1Fw5LihCDA"
                                }
                            ]
                        },
                        "nonce": "1",
                        "completion": "2021-04-07T14:53:36Z",
                        "amount": {
                            "0": {
                                "text": "28.2325781984DOT",
                                "currency": "DOT",
                                "numeric": 282325781984,
                                "exp": 10
                            }
                        },
                        "transfers": {
                            "1PLqR46mAv2uRPbdNtXqmQ869TfwS1H7m9Ku6yMeV5isnCd": [
                                {
                                    "account": {
                                        "id": "1PLqR46mAv2uRPbdNtXqmQ869TfwS1H7m9Ku6yMeV5isnCd"
                                    },
                                    "amounts": [
                                        {
                                            "text": "28.2325781984DOT",
                                            "currency": "DOT",
                                            "numeric": 282325781984,
                                            "exp": 10
                                        }
                                    ]
                                }
                            ]
                        },
                        "additional": {
                            "attributes": [
                                "{\"name\": \"AccountId\", \"value\": \"13n7RQQfwJ5iV34BjoFAi78Qt1RvN9pDuJBYDM1Fw5LihCDA\"}",
                                "{\"name\": \"AccountId\", \"value\": \"1PLqR46mAv2uRPbdNtXqmQ869TfwS1H7m9Ku6yMeV5isnCd\"}",
                                "{\"name\": \"Balance\", \"value\": 282325781984}"
                            ]
                        }
                    },
                    {
                        "id": "4530163-4",
                        "type": [
                            "deposit"
                        ],
                        "module": "balances",
                        "recipient": [
                            {
                                "account": {
                                    "id": "112A6wJPeDsf34nsqoAkAtQ8n74vJU8qmyKMtzX7ZPQH2kXa"
                                },
                                "amounts": [
                                    {
                                        "text": "0.0156000016DOT",
                                        "currency": "DOT",
                                        "numeric": 156000016,
                                        "exp": 10
                                    }
                                ]
                            }
                        ],
                        "node": {
                            "recipient": [
                                {
                                    "id": "112A6wJPeDsf34nsqoAkAtQ8n74vJU8qmyKMtzX7ZPQH2kXa"
                                }
                            ]
                        },
                        "nonce": "1",
                        "completion": "2021-04-07T14:53:36Z",
                        "amount": {
                            "0": {
                                "text": "0.0156000016DOT",
                                "currency": "DOT",
                                "numeric": 156000016,
                                "exp": 10
                            }
                        },
                        "transfers": {
                            "112A6wJPeDsf34nsqoAkAtQ8n74vJU8qmyKMtzX7ZPQH2kXa": [
                                {
                                    "account": {
                                        "id": "112A6wJPeDsf34nsqoAkAtQ8n74vJU8qmyKMtzX7ZPQH2kXa"
                                    },
                                    "amounts": [
                                        {
                                            "text": "0.0156000016DOT",
                                            "currency": "DOT",
                                            "numeric": 156000016,
                                            "exp": 10
                                        }
                                    ]
                                }
                            ]
                        },
                        "additional": {
                            "attributes": [
                                "{\"name\": \"AccountId\", \"value\": \"112A6wJPeDsf34nsqoAkAtQ8n74vJU8qmyKMtzX7ZPQH2kXa\"}",
                                "{\"name\": \"Balance\", \"value\": 156000016}"
                            ]
                        }
                    },
                    {
                        "id": "4530163-5",
                        "type": [
                            "extrinsicsuccess"
                        ],
                        "module": "system",
                        "nonce": "1",
                        "completion": "2021-04-07T14:53:36Z",
                        "additional": {
                            "attributes": [
                                "{\"name\": \"DispatchInfo\", \"value\": {\"weight\":202714000,\"class\":\"Normal\",\"paysFee\":\"Yes\"}}"
                            ]
                        }
                    }
                ]
            }
        ],
        "has_errors": false
    },
   ```

Comment (patiee) [reply README.md:40]: Yes ! You're right! Exp is 10

**Pull request merged**